ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/ImagesTableContainer.tsx

```diff
@@ -14,12 +14,15 @@ import { defaultImageSortFields, imagesDefaultSort } from '../sortUtils';
 import { DefaultFilters, VulnerabilitySeverityLabel, CveStatusTab } from '../types';
 import TableEntityToolbar from '../components/TableEntityToolbar';
 
+export { imageListQuery } from '../Tables/ImagesTable';
```

Comment:
This is re-exported since the parent component needs to refetch the query when the watched images list changes. Instead of directly importing the query from `../Tables/ImagesTable`, the re-export keeps the module dependency simple (`WorkloadCvesOverviewPage` -> `ImagesTableContainer`).